src/screens/Auth/LoginScreen/LoginScreen.tsx

```diff
@@ -2,15 +2,50 @@ import React from 'react'
 import {
   Text,
   View,
+  Alert,
+  Image,
+  TouchableOpacity
 } from 'react-native'
 import styles from './styles'
+import BlackHeading from '../../../shared_components/BlackHeading/BlackHeading';
+import GreyTextInput from '../../../shared_components/GreyTextInput/GreyTextInput';
+import PinkButton from '../../../shared_components/PinkButton/PinkButton';
+import PropTypes from 'prop-types'
+
+export default function LoginScreen(props) {
+  const {navigate} = props.navigation;
 
-export default function LoginScreen() {
   return (
     <View style={styles.container}>
-      <Text style={styles.title}>LoginScreen</Text>
+      <View style={styles.main}>
+        <BlackHeading title="Welcome Back" />
+        <Image style={styles.image} resizeMode='contain' source={require('../../../../assets/images/gladeo_logo.png')} />
+        <GreyTextInput placeholder="Email Address" inputType='emailAddress'/>
+        <GreyTextInput placeholder="Password" inputType='password'/>
+        <TouchableOpacity
+          onPress={() => navigate('PasswordReset')}
```

Comment:
Good use of arrow functions :) keep it up

Reply:
thanks! :)